src/main/resources/static/subjects-registration.js

```diff
@@ -2,41 +2,61 @@
  * Created by bo on 5/12/16.
  */
 
-var contains_patients_without_data;
-var contains_site_info_for_missing_paitents;
+var contains_site_info_for_missing_paitents = false;
+var template_str;
+var loading_html;
 
 $(document).ready(function () {
-    //TODO: backend checks
-    //for testing:
-    contains_patients_without_data = true;
-    contains_site_info_for_missing_paitents = false;
+    loading_html = '<div id="loading_div" class="loader"><br></div>';
+    $('#subject-registration-div').append(loading_html);
 
-    check_new_patients();
+    check_new_patients(true);
+});
+
+function check_new_patients(toRegisterSite) {
```

Comment:
this PR is about downloading patient registration template, validation is involved later in the uploading phase